Gemfile

```diff
@@ -33,8 +33,6 @@ group :development, :test do
   gem "faker"
   gem "govuk-content-schema-test-helpers"
   gem "govuk_test"
-  gem "jasmine"
-  gem "jasmine_selenium_runner"
```

Comment:
We should remove jasmine from: https://github.com/alphagov/collections-publisher/blob/301003f7bbfe437e7dda5413419c4dcd050b3b11/.github/dependabot.yml#L27-L28